src/components/molecules/NavList.tsx

```diff
@@ -1,53 +1,187 @@
-import React from 'react';
-import {List, ListItemButton, ListItemText, ListSubheader} from '@mui/material';
+import React, { useState } from "react";
+import {
```

Comment:
Can we compress this? I think the imports on one line is fine. I think VSCode dulls the imports that aren't being used anyway (unless thats a plugin I have) so it'll be easy to tell.

Reply:
Compressed and removed the one "dull" import.